reframe/schemas/runreport.json

```diff
@@ -143,16 +108,17 @@
             "items": {
                 "type": "object",
                 "properties": {
+                    "num_aborted": {"type": "number"},
                     "num_cases": {"type": "number"},
                     "num_failures": {"type": "number"},
-                    "num_aborted": {"type": "number"},
-                    "runid": {"type": "number"},
+                    "num_skipped": {"type": "number"},
+                    "runid": {"typeid": "number"},
```

Comment:
Yes, this is a typo. I changed also the `runid` to `run_index` as it's more precise and removed it from the test case since now test cases have their own UUID which contains the run and test case index.